packages/hub/process-controllers/hub-event-listener-controller.ts

```diff
@@ -0,0 +1,43 @@
+import { ContractSubscriptionEventHandler } from '../services/contract-subscription-event-handler';
+import { runInitializers, createRegistry } from '../main';
+import * as Sentry from '@sentry/node';
+import { Registry, Container } from '@cardstack/di';
+import { eventListenerLog } from '../utils/logger';
+
+export class HubEventListenerController {
```

Comment:
So the way the hub bot and worker controllers do their DI is actually inverted and we have tickets to fix them. that being said we should stop repeating this pattern. The idea is that these controllers should not create the container, rather the controllers live in the container. The correct pattern to follow is the HubServer. Where it is dependency injected into the container (rather than creating the container)--note how the HubServer is registered in the registry, and the CLI creates the container, and then grabs the HubServer out of the container and starts it. The Hub EventListener should follow this pattern too. it should have a CLI command that creates the container and then starts the hub event listener. You can refer to tickets #CS-2707 and #CS-2606